python/ray/rllib/tests/test_multi_agent_env.py

```diff
@@ -362,7 +362,8 @@ def testMultiAgentSampleAsyncRemote(self):
             policy_mapping_fn=lambda agent_id: "p{}".format(agent_id % 2),
             batch_steps=50,
             num_envs=4,
-            async_remote_worker_envs=True)
+            remote_worker_envs=True,
+            remote_worker_env_timeout_ms=0)
```

Comment:
Can you also add other test entries for timeout_ms > 0?

Reply:
Yes, fixed that.